### PR TITLE
feat(dev-doc): add doc on BPMN events execution

### PR DESCRIPTION
**Pull request by baptistemesta**

### CODE REVIEW

**Review by danila-m — CHANGES_REQUESTED** [review]
I know you simply copy pasted what we had in the wiki, but IMO it should not be merged like that.

- I'm really not sure we want the remarks in the official documentation.

- At least the empty sections should be removed (or ideally filled)

File: md/how-an-event-is-executed.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+## Message Trigger
+### Catch events
+### Throw events
+## Signal Trigger
+### Catch events
+### Throw events
+## Timer Trigger
+### Catch events
+### Throw events
```
Comment (baptistemesta) [inline md/how-an-event-is-executed.md]:
```suggestion
```

File: md/how-an-event-is-executed.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+
+## Events Type
+
+# Remarks
+
+* Error events, signals, messages etc. all go through the same code. It's suboptimal as they don't have much in common in their execution.
+* SWaitingErrorEvent (in db) seems redundant as to trigger it we already search in the process definition which elements catch the error.
+* Creating fake values to call handlePostThrowEvent() in case of an error event on a connector does not look clean code...
```
Comment (baptistemesta) [inline md/how-an-event-is-executed.md]:
```suggestion
```

**Comment by baptistemesta** [comment]
> I know you simply copy pasted what we had in the wiki, but IMO it should not be merged like that.
> 
> * I'm really not sure we want the remarks in the official documentation.
> * At least the empty sections should be removed (or ideally filled)

I agree, I updated the PR

**Review by danila-m — APPROVED** [review]

**Pull request merged**